prquantifier.yaml

```
@@ -0,0 +1,3058 @@
+Included: 
+#only include changes to files in source directory
+- /src
+Excluded:
+GitOperationType:
+- Add
+- Delete
+Thresholds:
+- GitOperationType:
+  - Add
+  - Delete
+  Value: 10
+  Label: Extra Small
+  Color: Green
+  Formula: Sum
+- GitOperationType:
+  - Add
+  - Delete
+  Value: 40
+  Label: Small
+  Color: Green
+  Formula: Sum
+- GitOperationType:
+  - Add
+  - Delete
+  Value: 200
+  Label: Medium
+  Color: DarkYellow
+  Formula: Sum
+- GitOperationType:
+  - Add
+  - Delete
+  Value: 400
+  Label: Large
+  Color: Red
+  Formula: Sum
+- GitOperationType:
+  - Add
+  - Delete
+  Value: 1000
+  Label: Extra Large
+  Color: Red
+  Formula: Sum
+LanguageOptions:
+  IgnoreSpaces: true
+  IgnoreComments: true
+  IgnoreCodeBlockSeparator: true
+DynamicBehaviour: false
+IgnoreRenamed: true
+IgnoreCopied: true
```

I'm not exactly sure what this means, but if someone copy/pastes something and then only changes 5% of it, the whole thing needs to be confirmed for correctness, so we probably want to include this in the size as well
```suggestion
IgnoreCopied: false
```